examples/autoqasm/6_Customize_gate_calibrations.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dda432a3",
+   "metadata": {},
+   "source": [
+    "# Customize gate calibrations\n",
+    "\n",
+    "Quantum hardware researches frequently require knowledge of native gate implementations on a specific devices, in the form of pulse sequences. The pulse implementation of gates is called \"gate calibrations\". In Amazon Braket, you have access to native gate calibrations provided by quantum hardware providers. Gaining access to gate calibrations empowers researchers to experiment with quantum gate designs and execute quantum programs with customized, improved gate calibrations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "30fd0b59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
```

Comment:
It might seem silly when writing, but I think it's important to _always_ introduce every code block. Make sure the reader is always expecting what is about to come next, and always knows what they should be looking out for or focusing on. This will make these resources effective for more of our customers